platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -97,12 +97,15 @@ protected void doInit() {
                         serviceContainerEnvironment))
             .collect(Collectors.toUnmodifiableList());
 
+    // Servers are processed in order. We want in process to be the last to start (so health check
+    // only returns when all other servers are up) and the last to shut down (so all other servers
+    // have drained external requests)
     this.servers =
         Stream.concat(
+                providedServers.stream(),
                 Stream.of(
                     new ConstructedServer(
-                        this.getInProcessServerName(), inProcessServerBuilder.build())),
-                providedServers.stream())
```

Comment:
You completely removed the providedServers?
I thought you were just changing order

Reply:
Sorry. The diff confused me.

Reply:
They moved to top of stream, line 105

Reply:
I was seeing the diff on the mobile app. It didn't show that.